README.md

```diff
@@ -200,6 +200,9 @@ Command flag              | Description
 `-s/--stdin`              | Pipe source from STDIN. This is useful for editor integration.
 `--[no-]color`            | Force color output on or off.
 
+Configuration options are loaded from `.rubocop`, command line switches, and the `RUBOCOP_OPTS` environment variable (listed in
```

Comment:
Configuration options may be taken from `.rubocop` file, command line options or environment variable. 

How can I rephrase it to be more clear?